pkg/descheduler/evictions/options.go

```diff
@@ -0,0 +1,45 @@
+package evictions
+
+import (
+	policy "k8s.io/api/policy/v1"
+)
+
+type Options struct {
+	policyGroupVersion         string
+	dryRun                     bool
+	maxPodsToEvictPerNode      *uint
+	maxPodsToEvictPerNamespace *uint
+	metricsEnabled             bool
+}
+
+// NewOptions returns an Options with default values.
+func NewOptions() *Options {
+	return &Options{
+		policyGroupVersion: policy.SchemeGroupVersion.String(),
```

Comment:
Just a though: I wonder whether there's ever going to be any other version than `policy/v1`. We detect the version at start up. We might as well drop it and assume `policy/v1` is always enabled. Re-introduce the ability to configure the version later.

Reply:
https://kubernetes.slack.com/archives/C18NZM5K9/p1720259403385159

Reply:
That sounds good, thanks for starting that!